src/main/java/use_case/translate/TranslateOutputData.java

```diff
@@ -9,4 +9,8 @@ public TranslateOutputData(String original, String translated) {
         this.original = original;
```

Comment:
Would adding the language attribute here be reasonable in case we'd like to extend it in the future , or could we create a different outputData class in that case? In my version, I included the language to relate it back to the entities, but if we move forward with a simpler version, we might not need to reference the entities since it would just be strings from yours and sophie's use case, and a list of strings from mine.